core/control-plane/control-plane-aggregate-services/src/test/java/org/eclipse/edc/connector/service/transferprocess/TransferProcessEventDispatchTest.java

```diff
@@ -76,14 +80,16 @@ void setUp(EdcExtension extension) {
         extension.registerServiceMock(EventExecutorServiceContainer.class, new EventExecutorServiceContainer(newSingleThreadExecutor()));
         extension.registerServiceMock(DataService.class, mock(DataService.class));
         extension.registerServiceMock(DataPlaneInstanceStore.class, mock(DataPlaneInstanceStore.class));
+        extension.registerServiceMock(PolicyArchive.class, mock(PolicyArchive.class));
     }
 
     @Test
     void shouldDispatchEventsOnTransferProcessStateChanges(TransferProcessService service, TransferProcessProtocolService protocolService,
-                                                           EventRouter eventRouter, RemoteMessageDispatcherRegistry dispatcherRegistry) {
+                                                           EventRouter eventRouter, RemoteMessageDispatcherRegistry dispatcherRegistry, PolicyArchive policyArchive) {
```

Comment:
Can you please clean-up the spacing to have the params on one line (if wide enough) or on each new line?

Reply:
Done with https://github.com/eclipse-edc/Connector/pull/2816/commits/a3b3562c84f48f57a1f0f522baefd3a063cb2081